crimsobot/cogs/reactions.py

```diff
@@ -0,0 +1,183 @@
+import logging
+
+from discord.ext import commands
+from discord.ext.commands import has_guild_permissions
+from tortoise.exceptions import DoesNotExist
+
+from config import ADMIN_USER_IDS
+from crimsobot.bot import CrimsoBOT
+from crimsobot.models.fun_facts import FunFacts, NoFactsExist
+from crimsobot.utils import tools as c
+
+log = logging.getLogger(__name__)
+
+
+class CleanMentions(commands.Converter):
+    async def convert(self, ctx: commands.Context, string: str) -> str:
+        """Clean up those silly mention inconsistencies across platforms."""
+        for mention in ctx.message.mentions:
+            string = string.replace(f'<@!{mention.id}>', f'<@{mention.id}>', 1)
+
+        return string
+
+
+class Reactions(commands.Cog):
+    def __init__(self, bot: CrimsoBOT):
+        self.bot = bot
+
+    @commands.group(aliases=['facts'])
+    async def fact(self, ctx: commands.Context) -> None:
+        """Testing out facts."""
+        return
```

Comment:
Instead of just returning here, it would probably be a good idea to send a message giving a brief explanation of how facts work and how the related subcommands work.

Reply:
I have aspirations here to have a "facts leaderboard" here, and will have added a `# TODO` for it in the next commit.